lib/api/core/models/repositories/roleRepository.js

```diff
@@ -239,6 +244,24 @@ class RoleRepository extends Repository {
 
     return serializedRole;
   }
+
+  _canLogin (role) {
+    for (let controllerKey of ['auth', '*']) {
+      if (role.controllers[controllerKey]) {
+        const controller = role.controllers[controllerKey];
+
+        for (let actionKey of ['login', '*']) {
+          if (controller.actions[actionKey]) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)